### DISCUSSION TITLE
Gleam on TIOBE index

### DISCUSSION THREAD

**Discussion by olorin37** [gleam-lang/gleam#3832] [General]
I found that TIOBE Index is not tracking this language. A pity as this is so vibrant that maybe could show up in first 100 :-) from time to time :) I'd proposed it to TIOBE via email.

**Comment by GearsDatapacks** [comment]
The TIOBE index isn't a very accurate measure of languages in my opinion. But I guess it could be good to get Gleam on there

**Reply by olorin37** [reply]
Actually I agree fully. Still it is a kind of reference. It measures _something_, but not what one would imagine looking at the index of the top programming languages :) But will be better to have Gleam acknowledged there.
